department-of-veterans-affairs/va.gov-cms
# VAGOV-4863: Dismissable alerts.

QA:
* Go to /admin/content then click on the blocks tab. It should appear as a table. 
* Edit a block, or add a new one.
* For banner alerts (formerly known as "full width alerts"), you should be able to specify whether or not they are dismissable. 
* The frequency field's help text should be updated to explain its relationship to dismissable.
